scripts/nolints.json

```diff
@@ -86,11 +86,16 @@
  ["docBlame", "termSudoSet_option___In_"],
  ["docBlame", "«term_\\_»"],
  ["docBlame", "«term_⁻¹»"],
+ ["docBlame", "«term_⁻¹'o_»"],
```

Comment:
Please document the syntax instead of silencing the lint.

Reply:
When I try to do that I get a linter error:
```
/- The `docBlame` linter reports:
DEFINITIONS ARE MISSING DOCUMENTATION STRINGS: -/
-- Mathlib.Order.Basic
#check term_⁻¹'o_ /- definition missing documentation string -/
```

Reply:
yes, because the definition is missing a doc string. You should fix it by adding one instead of silencing the lint.

Reply:
Aha, my problem was that I wasn't re-running `lake build`, I was just running the linter.

Reply:
Note: in recent versions of lean 4 core the `@[inheritDoc]` attribute exists to copy the doc string onto `infix` and `notation` declarations, and we will want to roll it out in mathlib4 along with the bump commit, which should fix this issue.